docs/polkadot/implementors-guide/guide.md

```diff
@@ -994,9 +992,44 @@ Dispatch a `PovFetchSubsystemMessage(relay_parent, candidate_hash, sender)` and
 
 (TODO: send statements to Statement Distribution subsystem, handle shutdown signal from candidate backing subsystem)
 
+### Candidate Selection Subsystem
+
+#### Description
+
+The Candidate Selection subsystem monitors net traffic for two events:
+
+- a new parablock candidate is available
+- a peer has seconded a parablock candidate
```

Comment:
The reason for this is that we'd prefer to avoid seconding the same parablock as another peer?

Given our short timeframe here, it might not be the case that higher candidate diversity is actually better.

Reply:
Yes, I'd had the impression that it was actually a slashable error to second the same parablock as a peer; the best we can do in that case is mark it as `Valid`.

Reply:
We do not afaik require slashing here.  see https://github.com/w3f/research/pull/87/files#r427365330